### PR TITLE
fc: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/windows/fc.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# fc
+
+> Compare the differences between two files or sets of files.
```
Comment (sbrl) [inline pages/windows/fc.md]: Compare the differences? Does this mean that it compares multiple diffs?

File: pages/windows/fc.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+> Use wildcards (*) to compare sets of files.
```
Comment (sbrl) [inline pages/windows/fc.md]: Define "sets of files"?

File: pages/windows/fc.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Compare two specified files:
```
Comment (sbrl) [inline pages/windows/fc.md]: `two` -> `2`?

File: pages/windows/fc.md (hunk 2 above)
Comment (pxgamer) [reply pages/windows/fc.md]: See the 3rd example here:
https://github.com/pxgamer/tldr/blob/master/pages/windows/comp.md

File: pages/windows/fc.md (hunk 1 above)
Comment (pxgamer) [reply pages/windows/fc.md]: This is the output from the command.

File: pages/windows/fc.md (hunk 2 above)
Comment (sbrl) [reply pages/windows/fc.md]: Ah, I see

**Comment by sbrl** [comment]
Merging due to inactivity.

**Pull request merged**